baselines/webworker.generated.d.ts

```diff
@@ -1663,30 +1663,35 @@ declare var XMLHttpRequest: {
 };
 
 interface XMLHttpRequestEventTargetEventMap {
-    "abort": Event;
-    "error": ErrorEvent;
-    "load": Event;
+    "abort": ProgressEvent;
+    "error": ProgressEvent;
+    "load": ProgressEvent;
     "loadend": ProgressEvent;
-    "loadstart": Event;
+    "loadstart": ProgressEvent;
     "progress": ProgressEvent;
     "timeout": ProgressEvent;
 }
 
-interface XMLHttpRequestEventTarget {
-    onabort: ((this: XMLHttpRequest, ev: Event) => any) | null;
```

Comment:
the `this` type was intentional see https://github.com/Microsoft/TypeScript/issues/15101 and https://github.com/Microsoft/TSJS-lib-generator/pull/275

Reply:
`XMLHttpRequestUpload` also extends `XMLHttpRequestEventTarget`, hmm. It doesn't have any properties so forcing XMLHttpRequest won't be an issue for now...

Reply:
Done